### PR TITLE
ADD unit test pkg/webhook

### PR DESCRIPTION
**Pull request by fewdan**
### What problem does this PR solve?
ADD unit test pkg/webhook

### What is changed and how does it work?
ADD unit test pkg/webhook

Tests

 - Unit test

Code changes

 - Has Go code change

Side effects

 - NO

Related changes

 - NO

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/mock/binding__failpoint_binding__.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package mock
+
```
Comment (zhouqiang-cl) [inline pkg/mock/binding__failpoint_binding__.go]: I think this file should not include

Comment (fewdan) [reply pkg/mock/binding__failpoint_binding__.go]: Do you mean not to submit to GitHub? @zhouqiang-cl

Comment (zhouqiang-cl) [reply pkg/mock/binding__failpoint_binding__.go]: yes

File: pkg/mock/mock.go [hunk 2]
```diff
@@ -55,9 +55,9 @@ var points = mockPoints{m: make(map[string]interface{})}
 // On inject a failpoint
 func On(fpname string) interface{} {
 	var ret interface{}
-	failpoint.Inject(fpname, func() {
+	if _, ok := failpoint.Eval(_curpkg_(fpname)); ok {
```
Comment (YangKeao) [inline pkg/mock/mock.go]: Please `make failpoint-disable`

Comment (fewdan) [reply pkg/mock/mock.go]: done

File: pkg/mock/binding__failpoint_binding__.go (hunk 1 above)
Comment (fewdan) [reply pkg/mock/binding__failpoint_binding__.go]: done

File: pkg/webhook/inject/setup_test.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+	err = scheme.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	err = chaosmeshv1alpha1.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	// +kubebuilder:scaffold:scheme
+
+	k8sManager, err = ctrl.NewManager(cfg, ctrl.Options{
+		Scheme: scheme.Scheme,
```
Comment (YangKeao) [inline pkg/webhook/inject/setup_test.go]: We can set `MetricsBindAddress: "0"` to disable metric server (which will listen :8080 by default)

Comment (YangKeao) [reply pkg/webhook/inject/setup_test.go]: Please also modify this for controller/suite_test.go

Comment (fewdan) [reply pkg/webhook/inject/setup_test.go]: done

**Pull request merged**